src/deepsparse/transformers/pipelines/haystack.py

```diff
@@ -0,0 +1,591 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
```

Comment:
let's move all haystack specific code into `deepsparse/transformers/haystack`, so we can add an `__init__.py` to dynamically install requirements